src/components/common/SelectInput/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
+import { arrayOf, bool, func, string } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { parseInputErrors } from 'utils/helpers';
+import Option from './Option';
+
+import '../../../styles/_variables.scss';
+
+const SelectInput = ({
+  label,
+  name,
+  value,
+  onChange,
+  errors,
+  active,
+  touched,
+  options,
+  ...props
+}) => {
+  // Register field in the form
+  useEffect(() => {
+    onChange({ target: { value } }, true);
```

Comment:
doesn't look pretty, but I think it's the way it's supposed to be used so it's ok 🤷‍♂️

Reply:
haha yes, apparently it was a fix made in the react-base repo so you could disable the submit button when there were input errors. I think there's a work-around this but it involves changing code in the useForm and useTextInputProps hooks, and it may be better to do so directly on the react-base repo.